src/sinks/influxdb_metrics.rs

```diff
@@ -0,0 +1,1056 @@
+use crate::{
+    event::metric::{Metric, MetricValue},
+    sinks::util::{
+        http::{Error as HttpError, HttpRetryLogic, HttpService, Response as HttpResponse},
+        BatchEventsConfig, MetricBuffer, SinkExt, TowerRequestConfig,
+    },
+    topology::config::{DataType, SinkConfig, SinkContext, SinkDescription},
+};
+use chrono::{DateTime, Utc};
+use futures::{Future, Poll};
+use http::{Method, StatusCode, Uri};
+use hyper;
+use hyper_tls::HttpsConnector;
+use lazy_static::lazy_static;
+use serde::{Deserialize, Serialize};
+use snafu::ResultExt;
+use std::cmp::Ordering;
+use std::collections::{BTreeMap, HashMap};
+use tower::Service;
+
+pub enum Field {
```

Comment:
Seems like this should include the other types listed on https://v2.docs.influxdata.com/v2.0/reference/syntax/line-protocol/#data-types-and-format ?

Reply:
I think that other types are unnecessary, but `Field` is useful for extending in future.